lua/catppuccin/groups/integrations/treesitter.lua

```diff
@@ -11,213 +11,118 @@ function M.get()
-			TSMethod = { fg = cp.blue, style = cnf.styles.functions or {} }, -- For method calls and definitions.
-			-- TSNone              = { };    -- TODO: docs
-			-- TSParameterReference= { };    -- For references to parameters of a function.
-			tomlTSProperty = { fg = cp.blue }, -- Differentiates between string and properties
-			TSPunctDelimiter = { fg = cp.overlay2 }, -- For delimiters ie: .
-			-- TSPunctBracket = { fg = delimeters }, -- For brackets and parenthesis.
-			TSPunctBracket = { fg = cp.overlay2 }, -- For brackets and parenthesis.
-			TSString = { fg = cp.green, style = cnf.styles.strings or {} }, -- For strings.
-			TSStringRegex = { fg = cp.peach, style = cnf.styles.strings or {} }, -- For regexes.
-			-- TSSymbol            = { };    -- For identifiers referring to symbols or atoms.
-			TSVariable = { fg = cp.text, style = cnf.styles.variables or {} }, -- Any variable name that does not have another highlighcp.
-			TSTagAttribute = { fg = cp.teal, style = { "italic" } }, -- Tags like html tag names.
-			TSTag = { fg = cp.mauve }, -- Tags like html tag names.
-			TSTagDelimiter = { fg = cp.sky }, -- Tag delimiter like < > /
-			TSText = { fg = cp.text }, -- For strings considerated text in a markup language.
-			-- TSEmphasis          = { };    -- For text to be represented with emphasis.
-			-- TSUnderline         = { };    -- For text to be represented with an underline.
-			-- TSStrike            = { };    -- For strikethrough texcp.
-			-- TSTitle             = { };    -- Text that is part of a title.
-			-- TSLiteral           = { };    -- Literal texcp.
-			-- TSURI               = { };    -- Any URI like a link or email.
-			--
-			-- Markdown tresitter parser support
-			TSURI = { fg = cp.rosewater, style = { "italic", "underline" } }, -- urls, links and emails
-			TSLiteral = { fg = cp.teal, style = { "italic" } }, -- used for inline code in markdown and for doc in python (""")
-			TSTextReference = { fg = cp.lavender, style = { "bold" } }, -- references
-			TSTitle = { fg = cp.blue, style = { "bold" } }, -- titles like: # Example
-			TSEmphasis = { fg = cp.maroon, style = { "italic" } }, -- bold
-			TSStrong = { fg = cp.maroon, style = { "bold" } }, -- italic
-			TSStringEscape = { fg = cp.pink, style = cnf.styles.strings }, -- For escape characters within a string.
-
-			-- bash
-			-- bashTSFuncBuiltin = { fg = cp.red, style = { "italic" } },
-			-- bashTSParameter = { fg = cp.yellow, style = { "italic" } },
-
-			-- json
-			jsonTSLabel = { fg = cp.blue }, -- For labels: label: in C and :label: in Lua.
-
-			-- lua
-			luaTSConstructor = { fg = cp.lavender }, -- For constructor calls and definitions: = { } in Lua, and Java constructors.
-
-			-- typescript
-			typescriptTSConstructor = { fg = cp.lavender },
-
-			-- TSX (Typescript React)
-			tsxTSConstructor = { fg = cp.lavender },
-			tsxTSTagAttribute = { fg = cp.mauve },
-
-			-- cpp
-			cppTSProperty = { fg = cp.rosewater },
-
-			-- yaml
```

Comment:
I meant restore this table so the fallback map actually works

Please test these changes on nvim 0.7.2 appimage